.github/workflows/virtual_box_rerun.yml

```diff
@@ -0,0 +1,35 @@
+# This is a basic workflow to help you get started with Actions
+
+name: virtual-box-functional-test-rerun
+
+# Controls when the workflow will run
+on:
+  workflow_run:
+    workflows: ["PR"]
+    branches: ["**"]
+    types: 
+      - completed
+
+env:
+  GH_TOKEN: ${{ github.token }}
+jobs:
+  check-and-rerun:
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: CheckArm64AndRerun
+        run: | 
+          set +e
+          gh run view ${{ github.event.workflow_run.id }} --log | grep ':type=>:arm'
```

Comment:
in the Original Job, we should have a Check that checks "step" for Architecture
and in the step it should fail Nice-er